Crm/Contact.php

```diff
@@ -701,6 +701,21 @@ public function toArrayForApi()
                 $return['custom_field_' . $fieldID] = $fieldValue;
             }
         }
+        if ($this->getBic()) {
+            $return['bic'] = $this->getBic();
+        }
+        if ($this->getFax()) {
+            $return['fax'] = $this->getFax();
+        }
+        if ($this->getGsm()) {
+            $return['gsm'] = $this->getGsm();
+        }
+        if ($this->getIban()) {
+            $return['iban'] = $this->getIban();
```

Comment:
This is not a valid option according to the api docs are you sure this works?
http://apidocs.teamleader.eu/crm.php